eras/shelley/impl/src/Cardano/Ledger/Shelley/Rules/Delegs.hs

```diff
@@ -215,15 +214,16 @@ delegsTransition = do
               (Map.mapKeys (mkRwdAcnt network) (UM.rewView (dsUnified ds)))
           )
 
-      let wdrls_' :: RewardAccounts (EraCrypto era)
-          wdrls_' =
+      let wdrls2 :: Map.Map (Credential 'Staking (EraCrypto era)) UM.RDPair
```

Comment:
maybe `drainedRewardAccounts` would be a better name?

Reply:
Good idea